web/source/dom/utils.ts

```diff
@@ -26,8 +26,8 @@ namespace com.keyman.dom {
         }
 
         // On mobile devices, the OSK uses 'fixed' - this requires some extra offset work to handle.
-        if(Lobj.style.position == 'fixed') {
-          let Ldoc = Lobj.ownerDocument;
+        let Ldoc = Lobj.ownerDocument;
+        if(Lobj.style.position == 'fixed' && Ldoc && Ldoc.scrollingElement) {
```

Comment:
I take it this is for compatibility with earlier versions of DOM?

Reply:
Yes - an error about how `scrollingElement` was undefined turned up in the console logs on some of the older APIs.